src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -297,7 +313,6 @@ def compute_accuracy(accuracy_metric, batch_loader):
         trainer.step(train_batch.data[0].shape[0])
         # calculate training metrics
         train_loss = loss.mean().asscalar()
-        train_time = _time.time() - start_time
```

Comment:
Why this change and the next two changes? Are you fixing another issue?

Reply:
In one of the previous PRs, I was suspicious the time elapsed is incorrectly calculated. The time printing earlier was for one mini batch of the epoch, this change correctly calculates time taken per epoch.